install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# A basic install wrapper for UNIX-based systems.
+#
+
+GITHUB_CHANGES_FILE="https://raw.githubusercontent.com/a-scie/lift/main/CHANGES.md"
+GITHUB_DOWNLOAD_BASE="https://github.com/a-scie/lift/releases/download"
+
+# Check if curl is available.
+if ! command -v curl &> /dev/null; then
+  echo "Error: curl command not found. Please install curl and try again." >&2
+  exit 1
+fi
+
+# Check if arch is available.
+if ! command -v arch &> /dev/null; then
+  echo "Error: arch command not found. Please install arch and try again." >&2
+  exit 1
+fi
+
+# Map platform architecture to released binary architecture.
+READ_ARCH="$(arch)"
+case "$READ_ARCH" in
+  x86_64*)   ARCH="x86_64" ;;
+  arm64*)    ARCH="aarch64" ;;
+  aarch64*)  ARCH="aarch64" ;;
+  *)        echo "unknown arch: $(READ_ARCH)"; exit 1 ;;
+esac
+
+# Map OS name to released binary OS names and platform-specific SHA checker invocation.
+case "$OSTYPE" in
+  darwin*)  OS="macos" SHASUM_CMD="shasum -a 256 -c -" ;;
+  linux*)   OS="linux" SHASUM_CMD="sha256sum -c -" ;;
+  *)        echo "unknown platform: ${OSTYPE}"; exit 1 ;;
+esac
+
+CHANGES_CONTENT=$(curl -s "${GITHUB_CHANGES_FILE}")
+if [ $? -ne 0 ]; then
+  echo "Error: Failed to fetch ${GITHUB_CHANGES_FILE} from $url" >&2
+  exit 1
+fi
+
+CURRENT_VERSION=$(echo "$CHANGES_CONTENT" | grep -m 1 '^#\+ [0-9]' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
+if [ -z "$CURRENT_VERSION" ]; then
+  echo "Error: No version number found in ${GITHUB_CHANGES_FILE}" >&2
+  exit 1
+fi
```

Comment:
No need for all this, in addition to `https://github.com/a-scie/lift/releases/download/v0.7.0/...` they support `https://github.com/a-scie/lift/releases/latest/download/...`.

Reply:
I thought so too, but didn't work for me on an initial test so I punted - will revert.